### ISSUE TITLE
Heading level is not defined for "Find the recipe for you" heading.

### ISSUE THREAD

**Issue by SuryacharanM** [dotnet/maui#6475]
GitHubTags:
#.NET MAUI; #A11ySev2; #AILimited; #A11yMAS; #A11yTCS; #Android; #AndroidPhone; #WCAG1.3.1; #GH_.NETMAUI_Android_Apr2022; #Headings​;

### Description

Heading level is not defined for "Find the recipe for you" heading.

Similar issue is observed throughout the application for all the headings.

### Steps to Reproduce

1. Install and open ".NET MAUI" app.
2. "Home Screen" will open.
3. Swipe till and observe the issue.

https://user-images.githubusercontent.com/86710788/165446955-bb31d40a-1c5c-46f8-831e-8180202c5c23.mp4

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 11

### Relevant log output

```shell
Heading level should be given for every heading.
```

**Comment by rachelkang** [comment]
Hi, @SuryacharanM - to my understanding, Android does not have a concept of "heading levels"; only Windows does. There is only the boolean of whether something is marked as a heading or not. Please let me know if I'm mistaken or misunderstanding something! Thanks :)
